code/modules/admin/DB ban/functions.dm

```diff
@@ -10,19 +10,19 @@ datum/admins/proc/DB_staffwarn_record(var/ckey, var/reason)
 	var/dbckey = sql_sanitize_text(ckey)
 
 	establish_db_connection()
-	if(!dbcon.IsConnected())
+	if(!SSdbcore.IsConnected())
 		to_chat(usr,"<span class='error'>Failed adding StaffWarn: db error</span>")
 		return
 
-	var/DBQuery/query = dbcon.NewQuery("SELECT id FROM erro_player WHERE ckey = '[dbckey]'")
+	var/datum/db_query/query = SSdbcore.NewQuery("SELECT id FROM erro_player WHERE ckey = '[dbckey]'")
 	query.Execute()
 	var/playerid = -1
 	if(query.NextRow())
 		playerid = query.item[1]
 	if(playerid == -1)
 		to_chat(usr,"<font color='red'>You've attempted to set staffwarn on [ckey], but they haven't been seen yet. Staffwarn can only be set on existing players.</font>")
 		return
-	query = dbcon.NewQuery("UPDATE erro_player SET staffwarn='[dbreason]' WHERE id=[playerid]")
+	query = SSdbcore.NewQuery("UPDATE erro_player SET staffwarn='[dbreason]' WHERE id=[playerid]")
 	query.Execute()
 	to_chat(usr,"<span class='notice'>StaffWarn saved to DB</span>")
```

Comment:
```suggestion
	to_chat(usr,"<span class='notice'>staffwarnsaved to database.</span>")
```

Reply:
You have a typo there

Reply:
The typo has become the typee....or something like that.

Reply:
:+1: